termination/termination.go

```diff
@@ -10,15 +10,12 @@ import (
 	"github.com/circleci/ex/o11y"
 )
 
-// ErrTerminated is used to indicate that the errgroup should cancel the
-// context, but that the shutdown is due to an expected signal, not an unhandled
-// error.
```

Comment:
These comments cover the callers behaviour  - not the behaviour of this package

e.g. it is up to the caller to decide if a signal is expected (and detect ErrTerminated)